common/core/core_api.c

```diff
@@ -96,6 +105,14 @@
 /*****************************************************************************
  * STATIC FUNCTIONS
  *****************************************************************************/
+void send_core_msg(core_msg_t *core_msg, const uint8_t *msg) {
+  uint8_t encoded_buffer[CORE_MSG_SIZE] = {0};
+  pb_ostream_t stream =
+      pb_ostream_from_buffer(encoded_buffer, sizeof(encoded_buffer));
+  ASSERT(pb_encode(&stream, CORE_MSG_FIELDS, core_msg));
+
+  usb_send_msg(encoded_buffer, stream.bytes_written, msg, 0);
```

Comment:
this seems wrong. the size of `msg` is `0`.

Reply:
updated